app/models/item.rb

```diff
@@ -1,4 +1,14 @@
 class Item < ActiveRecord::Base
   validates :name, presence: true
-  validate :description, presence: true
+  validates :description, presence: true
+
+  belongs_to :user
+
+  has_attached_file :image, styles: {
+    thumb: '100x100>',
+    square: '200x200#',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.